models/typescript/index.d.ts

```diff
@@ -0,0 +1 @@
+export * from './v3.0';
```

Comment:
Even I see the benefit of having an `index.d.ts` file. However, we will have name conflicts across versions. One possible solution is re-exporting the package with an alias like:
```
import * as v3 from './v3.0';
import * as v2 from './v2.0';
export { v3, v2 }
```